claudio-plugin/skills/jira-utilities/tests/test_ansi_stripping.py

```
@@ -0,0 +1,75 @@
+        capture_output=True,
+        text=True,
+        env=os.environ.copy(),
+    )
+    assert result.returncode == 0, f"get_issue.sh failed: {result.stderr}"
+    data = json.loads(result.stdout)
+    assert data["key"] == "AIPCC-7229"
+    assert data["fields"]["summary"] == "Test issue for ANSI stripping"
+
+
+def test_get_issue_no_ansi_in_output(acli_ansi_view):
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)